frontend/www/js/omegaup/components/qualitynomination/List.vue

```diff
@@ -120,14 +120,14 @@ import { Vue, Component, Prop, Emit, Watch } from 'vue-property-decorator';
 import { omegaup } from '../../omegaup';
 import T from '../../lang';
 import * as UI from '../../ui';
-import paginador from '../common/Paginatorv2.vue';
+import paginator from '../common/Paginatorv2.vue';
```

Comment:
ah! sabes qué? aquí me faltó algo, esto debería ser

```suggestion
import common_Paginator from '../common/Paginatorv2.vue';
```

><